spring-cloud-app-broker-deployer/src/main/java/org/springframework/cloud/appbroker/deployer/CreateServiceKeyRequest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.appbroker.deployer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.util.CollectionUtils;
+
+public class CreateServiceKeyRequest {
+
+	private final String serviceInstanceName;
+
+	private final String serviceKeyName;
+
+	private final Map<String, Object> parameters;
+
+	private final Map<String, String> properties;  // to access space target property
```

Comment:
Please, delete comment to be consistent with the other methods.

Reply:
fixed in 10fd24ea9ebc1ce01db90c1eee8a3e2a089fb46d